README.md

```diff
@@ -85,6 +85,63 @@ NavigationStackView(transitionType: .custom(AnyTransition.scale.animation(.sprin
 
 attaching the easing directly to the transition. **Don't do this**. SwiftUI has still some problems with implicit animations attached to transitions, so it may not work. For example, implicit animations attached to a .slide transition won't work.
 
+## NavigationStack injection
```

Comment:
Let's move this entire section between the `Pop` section and the `Important` section because I've just realised we only mention the `NavigationStack` (and the programmatic navigation) in a section below. Here it's still confusing.